cli/smartcontract/smart_contract.go

```diff
@@ -387,12 +398,13 @@ func NewCommands() []cli.Command {
 						Action: manifestAddGroup,
 						Flags: []cli.Flag{
 							walletFlag,
+							walletConfigFlag,
 							cli.StringFlag{
 								Name:  "sender, s",
 								Usage: "deploy transaction sender",
 							},
-							cli.StringFlag{
-								Name:  "account, a",
+							flags.AddressFlag{
+								Name:  addressFlagName, // use the same name for handler code unification.
```

Comment:
Breaks backwards compatibility somewhat, but they're the same in short form and it's a very rarely used feature (and the only command that differs), so probably OK.